ChaosMod/Effects/db/Vehs/VehsDamageController.cpp

```diff
@@ -0,0 +1,86 @@
+#include "stdafx.h"
+
+// clang-format off
```

Comment:
Why?

Reply:
I think at the time it was screwing with something in my compiler. I must've forgotten to remove it.